src/hidimstat/base_variable_importance.py

```diff
@@ -167,7 +170,11 @@ def plot_importance(
 
         if ax is None:
             _, ax = plt.subplots()
-        feature_names = list(self.groups.keys())
+
+        if hasattr(self, "feature_groups"):
+            feature_names = list(self.feature_groups.keys())
+        else:
+            feature_names = [str(j) for j in range(self.importances_.shape[-1])]
```

Comment:
For any class inheriting `GroupVariableImportanceMixin`, hasattr(self, "feature_groups")=True. Can we instead: 
 - Have a parameter feature_names
 - if feature_names is None and self.feature_groups is None: feature_names = list(self.feature_groups.keys())
 - else: feature_names = [str(j) for j in range(self.importances_.shape[-1])]

Reply:
done